R/install_torsten.R

```diff
@@ -0,0 +1,197 @@
+#' Install Torsten
+#'
+#' @description The `install_torsten()` function attempts to download and
+#'   install the latest release of [Torsten](https://github.com/metrumresearchgroup/Torsten/releases/latest).
+#'   Installing a previous release or a new release candidate is also possible
+#'   by specifying the `version` or `release_url` argument.
+#'   See the first few sections of the CmdStan
+#'   [installation guide](https://mc-stan.org/docs/cmdstan-guide/cmdstan-installation.html)
+#'   for details on the C++ toolchain required for installing CmdStan.
+#'
+#'
+#' @export
```

Comment:
This docstring needs to be above `install_torsten` so that it's exported. As it is now, it exports `latest_torsten_release`.

As a related but more general note, the documentation needs to be regenerated and the `NAMESPACE` file updated. This can be done by calling `devtools::document()`.

Reply:
I moved `latest_torsten_release` and `get_torsten_release_list` to the end. They will remain unexported. 

Thanks for the tip on generating the documentation and `NAMESPACE`.